### DISCUSSION TITLE
LCD grid on GBA games...but in RetroArch?

### DISCUSSION THREAD

**Discussion by chrisstonehill** [eduardofilo/RG350_adam_image#161] [RetroArch topics]
Ah ha...maybe I have just found out myself that for GBA games I seem to need to choose the REGBA "core? (is that the right term...perhaps emulator) instead of the Retroarch ones, one of them being the default...if I want the LCD grid...that I think looks better, and seemingly all of the other handhelds have WITH the default Retroarch emulators...

...or is there a setting that I can't see when trying GBA games with Retroarch? I choose Options from the Retroarch menu but can't see anything there. Under that is "On-Screen Overlay", but that doesn't seem to be it. Would be nice to maintain the Retroarch consitency (hotkeys for example)...but the image is a bit fuzzy, I find, without that LCD grid.

**Comment by eduardofilo** [comment]
ReGBA is said to be a standalone emulator.
The other options configured for GBA in SimpleMenu are two RetroArch cores: gpSP and mGBA.
Those RA cores can display grids as you want, but because GBA is a difficult platform to emulate, activating those effects often cause slowdowns.
If you want to try, the grids are activated through video filters. Look for them in `MAIN MENU > Settings > Video > Filters` (I speak from memory).

**Reply by chrisstonehill** [reply]
Thanks again Eduardo. Found it! I had seen before, but forget when looking again recently that there are further RetroArch menu pages to explore, other than the one that pops up when you push the power button. LOTS of grid filters to choose from!!!...but if they often cause slowdown, maybe I'll just keep changing to ReGBA for all GBA games I want to play.

Just been thinking what a good position I am in to be experiencing all of this emulation, due to the fact that most of the systems I have no experience of the original, just would have liked to have had those experiences. I had a handful of games on our family Atari ST and a similar, slim amount on a original Gameboy that I bought from a friend at the end of primary school, at the start of the 90s I guess. Played a tiny bit on another friends SNES. Vaguely remember his C64 and NES. I think another friend had a BBC micro...and a little bit of (mainly observing) Wolfenstein and DOOM on another friend's PC. Only PC game I bought back then was Quake 2, but my interest was shifting to other things, so barely played it. Occasionally saw arcade games (and thought they looked AWESOME) but I don't think I ever played any. However, all of this means that almost all inaccuracies in emulation are lost on me...apart from audio on my AtGames handheld Megadrive thing :-) How incredibly bad and off key that often becomes is glaringly apparent :-D
